fstest/fstests/gen_tests.go

```diff
@@ -66,7 +66,7 @@ import (
 	"github.com/ncw/rclone/fs"
 	"github.com/ncw/rclone/fstest/fstests"
 	"github.com/ncw/rclone/{{ .FsName }}"
-{{ if eq .FsName "crypt" }}	_ "github.com/ncw/rclone/local"
+{{ if (or (eq .FsName "crypt") (eq .FsName "cache")) }}	_ "github.com/ncw/rclone/local"
```

Comment:
@ncw just want to flag this to your attention cause it's kind of outside the standard changes. Since both `cache` acts as a wrapper I had to add it to this `if` to have that import added.

Reply:
That looks fine - cache and crypt are similar in that respect.